packages/core/src/utils/ConstantsUtil.ts

```diff
@@ -61,7 +61,99 @@ export const ConstantsUtil = {
     Base: 'base'
   },
 
-  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e'
+  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e',
+
+  SUGGESTED_TOKENS: [
+    'ETH',
+    'UNI',
+    '1INCH',
+    'AAVE',
+    'SOL',
+    'ADA',
+    'AVAX',
+    'DOT',
+    'LINK',
+    'NITRO',
+    'GAIA',
+    'MILK',
+    'TRX',
+    'NEAR',
+    'GNO',
+    'WBTC',
+    'DAI',
+    'WETH',
+    'USDC',
+    'USDT',
+    'ARB',
+    'BAL',
+    'BICO',
+    'CRV',
+    'ENS',
+    'MATIC',
+    'OP'
+  ],
+
+  POPULAR_TOKENS: [
+    'ETH',
+    'UNI',
+    '1INCH',
+    'AAVE',
+    'SOL',
+    'ADA',
+    'AVAX',
+    'DOT',
+    'LINK',
+    'NITRO',
+    'GAIA',
+    'MILK',
+    'TRX',
+    'NEAR',
+    'GNO',
+    'WBTC',
+    'DAI',
+    'WETH',
+    'USDC',
+    'USDT',
+    'ARB',
+    'BAL',
+    'BICO',
+    'CRV',
+    'ENS',
+    'MATIC',
+    'OP',
+    // Some Polygon tokens
+    'DAI',
```

Comment:
For testing, I've whitelisted some tokens to list in the token select page, we shouldn't trim the token array. Since it's too big list, the performance of the token select dropping. Waiting the Lit team to merge this: https://github.com/lit/lit/pull/4564. Then we could render large lists, then we don't need to filter the tokens